src/xp_pen_handler.cpp

```diff
@@ -50,6 +51,7 @@ xp_pen_handler::xp_pen_handler() {
     addHandler(new artist_22r_pro());
     addHandler(new artist_22e_pro());
     addHandler(new artist_16_pro());
+	addHandler(new artist_pro_16tp());
```

Comment:
nitpick: Looks like tab vs spaces here. Its fine, I can always format it at some point later